framework/tlvf/src/include/tlvf/CmduMessage.h

```diff
@@ -44,16 +42,24 @@ class CmduMessage {
     // Forward wrapper functions
     // TODO check which of them can be removed
     template <class T> std::shared_ptr<T> getClass() const { return msg.getClass<T>(); };
+    template <class T> std::list<std::shared_ptr<T>> getClassList() const
+    {
+        return msg.getClassList<T>();
+    };
     size_t getMessageLength() const { return msg.getMessageLength(); };
     size_t getMessageBuffLength() const { return msg.getMessageBuffLength(); };
     uint8_t *getMessageBuff() const { return msg.getMessageBuff(); };
+    template <class T> std::shared_ptr<T> getClass() { return msg.getClass<T>(); }
```

Comment:
There is already a const getClass version a bit higher up, why do we also need a non-const one?

Reply:
we don't, nice catch.